README.md

```diff
@@ -0,0 +1,52 @@
+# holo-auto-pilot
+
+## Usage
+
+```
+$ holo-auto-pilot --help
+USAGE:
+    hpos-holo-auto-pilot [OPTIONS] <happ-list-path> <membrane-proofs>
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+        --admin-port <admin-port>              Holochain conductor port [env: ADMIN_PORT=]  [default: 4444]
+        --happ-port <happ-port>                hApp listening port [env: HAPP_PORT=]  [default: 42233]
+
+ARGS:
+    <happ-list-path>    Path to a YAML file containing the list of hApps to install
+    <membrane-proof>    Path to a YAML file containing the list of mem_proof that is used to install
```

Comment:
would be good to have an example of the membrane proof file format in this README also

Reply:
Afaict, the same membrane proofs are used for installing every happ. It's probably worth calling that out here as well. I'm assuming in the future this won't be the case?

Reply:
Sounds good. I did not focus on the readme too much. Tks for the catch :)